cPP_DSC.adoc

```diff
@@ -1023,6 +1023,94 @@ FCS_CKM.2.1:: The TSF shall distribute cryptographic keys in accordance with a s
 
+==== FCS_CKM_EXT.3 Cryptographic Key Access
+
+FCS_CKM_EXT.3 Cryptographic Key Access
+
+FCS_CKM_EXT.3.1:: The TSF shall use specified cryptographic key access methods [*selection*: _key encapsulation, key wrapping, key encryption_] that meets the following: [*selection*: _list of standards that provide integrity and confidentiality_] to access keys when performing [*selection*: _cryptographic key usage in cryptographic operations, cryptographic key archival, cryptographic key backup, cryptographic key escrow, cryptographic key recovery, modifications to attributes of cryptographic keys, cryptographic key destruction_].
+
+_Application Note {counter:remark_count}_:: 
++
+*_Key Access Operations_*
++
+_There are several reasons for granting access to keys or restricting access to keys.  The list below enumerates typical actions for keys in a typical lifecycle of a key.  Further on in this note are listed some methods used to authenticate roles and to authorize them to perform these actions._
++
+_The typical actions for the lifecycle of a key may look something like:_
++
+--
+. _Creation_
+. _Use in cryptographic operation_
+. _Archival, Backup and Escrow_
+. _Recovery_
+. _Attribute modification_
+. _Destruction_
+--
++
+*_Creation of cryptographic keys* - It may be the case that during the creation of the key certain attributes are assigned to it that controls future access to the key, such as who is allowed to use it or restricted from using it, when they are allowed to use it or restricted from using it, who is allowed to backup and recover keys, who is allowed to destroy them, and who is allowed to modify attributes associated with the keys.  This SFR imposes no requirements on the creation of cryptographic keys, other than those specified already in FCS_CKM.1.  However, once created, this SFR does impose requirements to access keys for the purposes outlined in the first assignment._
++
+*_Cryptographic key usage in cryptographic operations* - Depending on the application, a cryptographic key could be root key which by virtue of being a root key, cannot be encrypted with another key.  It is often stored in plain sight, sometimes with hardware protections, sometimes obfuscated in clever ways, such as in physically unclonable functions.  Other than root keys, it is expected that other cryptographic keys should be protected with encryption, often using other keys, which leads to a chain or hierarchy of cryptographic keys that encrypt other keys often anchored by a root key.   Additionally, it is expected that an entity authenticate itself self as an authorized user of the key.  This latter requirement is in addition to the recommended requirements of the Common Criteria Part 2 for FCS_CKM.3._
```

Comment:
Regarding the last sentence: What exactly should that statement imply - any special actions for the ST author?